src/PartialModelSelection.cpp

```diff
@@ -13,30 +15,34 @@
 /*MODEL SELECTION MAP IMPLEMENTATIONS*/
 
 //Default contructor for model selection map, sets max models to std of 3.
```

Comment:
default should be no max

Reply:
Right, I think I set it to infinity as a default value to condense the code to one constructor. I forgot to remove the unused default constructor, will fix.